app/views/users/index.html.erb

```diff
@@ -15,6 +15,8 @@
         <td><%= user.email %></td>
         <td><%= user.is_male ? 'male' : 'female' %></td>
         <td><%= user.birthday %></td>
+        <td><%= user.interests.map { |interest| interest.name }.join(', ')  %></td>
```

Comment:
`user.interests.map` results in the following query:
```SQL
SELECT "interests".* FROM "interests" WHERE "interests"."user_id" = ?
```
In other words, it:
- fetches all interests data (interests.*),
- for each of them, it instantiates a separate active record object,
- then it takes only `name` from it
-  and then it disregards those AR objects as not needed anymore.

A much more performant option would be to use the following query:
```SQL
SELECT "interests"."name" FROM "interests" WHERE "interests"."user_id" = ?
```
So, to fetch only needed data (names), and use them without instantiating unnecessary AR objects.
[You can use `pluck` for that](https://apidock.com/rails/ActiveRecord/Calculations/pluck).